sphinx/writers/latex.py

```diff
@@ -373,8 +375,8 @@ def __init__(self, document: nodes.document, builder: "LaTeXBuilder",
         if (self.config.language not in {None, 'en', 'ja'} and
                 'fncychap' not in self.config.latex_elements):
             # use Sonny style if any language specified (except English)
-            self.elements['fncychap'] = ('\\usepackage[Sonny]{fncychap}\n'
-                                         '\\ChNameVar{\\Large\\normalfont\\sffamily}\n'
+            self.elements['fncychap'] = ('\\usepackage[Sonny]{fncychap}' + CR +
+                                         '\\ChNameVar{\\Large\\normalfont\\sffamily}' + CR +
                                          '\\ChTitleVar{\\Large\\normalfont\\sffamily}')
```

Comment:
This change allows us to use raw strings (`r'...'`) in the LaTeX writer. I believe it makes writing TeX macros simple.

Reply:
+1 for idea! If idea goes into tests/test_build_latex.py we will have initial work to unify, sometimes raw strings are used, but mostly note. Raw strings make it easier to check what happens in case of test failures and to adapt tests if things change.